user/views.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from rest_framework import generics, permissions
+from rest_framework.authentication import TokenAuthentication
+from rest_framework.authtoken.views import ObtainAuthToken
+from rest_framework.settings import api_settings
+
+from user.serializers import UserSerializer
+
+
+class CreateUserView(generics.CreateAPIView):
```

Comment:
According to the checklist, it is required to add a blank line at the end of your files. Please add a blank line at the end of this file.